CondTools/HLT/test/AlCaRecoTriggerBitsRcdUpdate_cfg_o2o.py

```diff
@@ -0,0 +1,147 @@
+                  VarParsing.VarParsing.multiplicity.singleton, 
+                  VarParsing.VarParsing.varType.string,
+                  "the input tag"
+                  )
+
+options.register( "outputDB", 
+                  "sqlite_file:AlCaRecoTriggerBits.db",  #default value
+                  VarParsing.VarParsing.multiplicity.singleton, 
+                  VarParsing.VarParsing.varType.string,
+                  "the output DB"
+                  )
+
+options.register( "outputTag", 
+                  "AlCaRecoTriggerBitsTag",  #default value
+                  VarParsing.VarParsing.multiplicity.singleton, 
+                  VarParsing.VarParsing.varType.string,
+                  "the output tag"
+                  )
+
+options.register( "firstRun", 
+                  1,  #default value
+                  VarParsing.VarParsing.multiplicity.singleton, 
+                  VarParsing.VarParsing.varType.int,
+                  "the first run"
+                  )
+
+options.register( "hltKey", 
+                  "/cdaq/special/commissioning2021/CRAFT/Cosmics/V4",  #default value
+                  VarParsing.VarParsing.multiplicity.singleton, 
+                  VarParsing.VarParsing.varType.string,
+                  "the hlt key"
+                  )
+
+options.register( "keyToModify", 
+                  "SiStripCalMinBias",  #default value
+                  VarParsing.VarParsing.multiplicity.singleton, 
+                  VarParsing.VarParsing.varType.string,
+                  "the key to modify"
+                  )
+
+options.register('pathsToModify',
+                 'HLT_*ZeroBias_part*_v*', #default value
+                 VarParsing.VarParsing.multiplicity.singleton,
+                 VarParsing.VarParsing.varType.string,
+                 "Comma-separated list of paths to be modified")
+options.parseArguments()
+
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+process.MessageLogger.cerr = cms.untracked.PSet(enable = cms.untracked.bool(True))
+process.MessageLogger.cout = cms.untracked.PSet(INFO = cms.untracked.PSet(
+    reportEvery = cms.untracked.int32(1)
+    ))
+process.MessageLogger.cout.enable = cms.untracked.bool(True)
+process.MessageLogger.cout.threshold = cms.untracked.string('DEBUG')
+process.MessageLogger.debugModules = cms.untracked.vstring('*')
+# the module writing to DB
+from CondTools.HLT.alCaRecoTriggerBitsRcdUpdate_cfi import alCaRecoTriggerBitsRcdUpdate as _alCaRecoTriggerBitsRcdUpdate
+process.AlCaRecoTriggerBitsRcdUpdate = _alCaRecoTriggerBitsRcdUpdate.clone()
+# The IOV that you want to write out, defaut is 1 to -1/inf. 
+process.AlCaRecoTriggerBitsRcdUpdate.firstRunIOV = options.firstRun # docu see...
+#process.AlCaRecoTriggerBitsRcdUpdate.lastRunIOV = -1 # ...cfi
+# If you want to start from scratch, comment the next line:
+process.AlCaRecoTriggerBitsRcdUpdate.startEmpty = False
+
+# add paths if hlt key has 'special', remove those for the other cases
```

Comment:
I personally don't have any historical perspective on this (and far from me to question this choice), but I wonder if TSG/FOG is fully aware that subsystems rely on the name of the HLT config to make decisions downstream. Might be worth a reminder, imho (fyi: @silviodonato @mzarucki). As described in the PR, this is explained [here](https://indico.cern.ch/event/1041042/contributions/4373068/attachments/2249069/3815192/TriggerBitO2OforSiStrip.pdf).

Reply:
This is actually a very good point that I have already made with @ggovi.
The selection of runs to trigger the O2O, as is too now, is too inclusive.
When I wrote "special" in the [deck of slides](https://indico.cern.ch/event/1041042/contributions/4373068/attachments/2249069/3815192/TriggerBitO2OforSiStrip.pdf) pointed above, I meant a generic "special" and not that the name of the hlt key **literally** contained the string `special`. 
I did a bit of research based off this [twiki page](https://twiki.cern.ch/twiki/bin/viewauth/CMS/SiStripSignalHVScans) and organized the information about the HLT menu used to acquire all the SiStrip Bias Scan runs in Run2 in [this gist](https://gist.github.com/mmusich/f12f1681d819ab52ad74796f185f1641).
Reporting here only the Full scans (which is what this PR is targeting, as for the Small ones, it's impossible to tell just from the HLT menu):

![image](https://user-images.githubusercontent.com/5082376/156934037-64b04e6d-73a9-403b-8b6d-c4d8b0718502.png)

one can see that actually most of the times the HLT key didn't contain the string `special` at all, excepted in the very last few runs of 2018. I cannot say if this (convenient) policy will remain in place.
I think a safer approach - which I also tried to highlight during the meeting with the AlCa/DB group - would be instead to grab the menu, dump it and look if a partitioned `ZeroBiasX [X=1,...N]` set of triggers exists. This is still not perfect as it would trigger the O2O also in some other cases, but at least would rely less on the HLT key nomenclature, unless - of course - the TSG group agrees to use some specific key to capture the scans.

Reply:
Thanks for the info, Marco. As for the naming conventions of online menus, and how this can be coordinated with TSG, I defer to Mateusz, Silvio and Darin (might be worth discussing it with them by email or in a relevant meeting).

Reply:
Thank you for the information. In fact, this was the part I wanted to be confirmed/discuss along with the PR. Afaik there can be other cases that partitioned triggers included other than the Full bias scan, so indeed it would be safer if the scan can be somehow specified from the menu. 
IMHO, we don't need to rush for `12_3_0_pre6` as local patch in o2o machine is very doable.(we're already doing it for test) (besides I'll be slow till 09/Mar(analysis approval talk))

Reply:
Hi, from the FOG perspective we strongly discourage that subsystems would rely on the names of HLT menus to make decisions downstream. It can be rather dangerous, as naming schemes and conventions might change over time. 

Nevertheless, we are currently in the process of improving our internal conventions and making them more transparent for Run 3 - while the `cosmic` and `physics` categories have followed stricter conventions in the past, the `special` one historically did not. Of course, we would be open to receive any inputs for this procedure, but this would have to be discussed in an e-mail/JIRA thread or one of the TSG meetings (as suggested by @missirol). However, I still must underline the first argument.

Reply:
>  Of course, we would be open to receive any inputs for this procedure, but this would have to be discussed in an e-mail/JIRA thread or one of the TSG meetings (as suggested by @missirol). However, I still must underline the first argument.

I'd really rather let @hjkwon260 or @cms-sw/alca-l2 to follow-up on that, since they're supposed to communicate with TSG about trigger needs from subsystems in terms of  triggers.
On the other hand given:

> we strongly discourage that subsystems would rely on the names of HLT menus to make decisions downstream.

I would would encourage @hjkwon260 to follow my suggestion at https://github.com/cms-sw/cmssw/pull/37119#discussion_r820262938 about:

> to grab the menu, dump it and look if a partitioned `ZeroBiasX [X=1,...N]` set of triggers exists.

Finally about: 

> IMHO, we don't need to rush for 12_3_0_pre6 as local patch in o2o machine is very doable.

I'd like to ask @ggovi's confirmation, as it doesn't seem a very safe policy.

Reply:
@malbouis honestly I have no idea about what DAQ flag you are talking about. I would invite you to investigate about it, if you know what would be necessary.

Reply:
I thought maybe the Strips DAQ would include some information about the HV scan in the event. But from your comment such a thing probably does not exist. I will try to investigate though. What I am trying to think about is which other kind of information could we use to combine with the trigger paths one? So I'm trying to think about options. ;-)